src/CloudStructures/Internals/EnumerableExtensions.cs

```diff
@@ -50,7 +50,7 @@ public static IEnumerable<T> Materialize<T>(this IEnumerable<T>? source, bool nu
         if (source is null)
         {
             if (nullToEmpty)
-                return [];
+                return new T[0];
```

Comment:
What's this?

Reply:
The code was not compiling since it gave errors on returning an empty array like so. 
So, I google searched and found this solution of returning an empty array